.github/workflows/test_be.yaml

```diff
@@ -116,13 +116,13 @@ jobs:
       - name: Test with minimal dependencies
         if: ${{ matrix.dependencies == 'core' }}
         run: |
-          hatch run +py=${{ matrix.python-version }} test:test -v tests/ -k "not test_cli" --durations=10
+          hatch test -py ${{ matrix.python-version }} -v -k "not test_cli" --durations=10
```

Comment:
i didn't use hatch's test env because it diverges from `test-optional`. if there is a way to run `hatch test` with the optional deps, then happy to move to hatch's built-in test env, otherwise i think the previous way is clearer.